docs/docs/how-to/submit-transaction.md

```diff
@@ -47,10 +47,10 @@ cardano-cli transaction sign \
 cat tx-signed.json | jq -c '{tag: "NewTx", transaction: .}'
 ```
 
-This will output a message that can be submitted to a websocket connection to `hydra-node`. Using `websocat` and assuming the `hydra-node` runs on the default port `4001` we can submit with:
+This command generates a message suitable for submission to the `hydra-node` via a WebSocket connection. If `hydra-node` operates on the default port `4001`, the message can be submitted using `websocat`:
 
 ```shell
 cat tx-signed.json | jq -c '{tag: "NewTx", transaction: .}' | websocat "ws://127.0.0.1:4001?history=no"
 ```
 
-The transation will now be validated by all connected `hydra-node`s and either result in a `TxInvalid` message with a reason, or a `TxValid` message and a `SnapshotConfirmed` with the new UTxO available in the head shortly after.
+The transaction will be validated by all connected `hydra-node` instances. It will result in either a `TxInvalid` message, providing a reason for rejection, or a `TxValid` message followed by a `SnapshotConfirmed`, updating the UTXO available in the Head shortly thereafter.
```

Comment:
```suggestion
The transaction will be validated by all connected `hydra-node` instances. It will result in either a `TxInvalid` message, providing a reason for rejection, or a `TxValid` message followed by a `SnapshotConfirmed`, updating the UTXO available in the head shortly after that.
```